gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicationLocation.java

```diff
@@ -0,0 +1,12 @@
+package gobblin.data.management.copy.replication;
+
+/**
+ * Interface ReplicationLocation is used to specify the location of a replication data
+ * 
+ * @author mitu
+ *
+ */
+public interface ReplicationLocation {
```

Comment:
I don't understand this interface. There is nothing that describes an actual location.